pkg/probe/exec/errors_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+import (
+	"errors"
+	"testing"
+	"time"
+)
+
```

Comment:
We can combine the 3 tests into one, then we can extend them more easily, like the following.

https://github.com/kubernetes/kubernetes/blob/4211826c3c835ab6940e5af536e01434feac8d30/pkg/probe/exec/exec_test.go#L103-L148

Reply:
Okay I will do this.